resources/views/posts/edit-add.blade.php

```diff
@@ -102,7 +102,7 @@
                     <!-- ### CONTENT ### -->
                     <div class="panel">
                         <div class="panel-heading">
-                            <h3 class="panel-title"><i class="icon wb-book"></i> {{ __('voyager::voyager.post.content') }}</h3>
```

Comment:
The icon `wb-hook` is from way before the `v1.0` release and ends up with given extra un-needed space.